src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -14,8 +14,10 @@ if(CLR_CMAKE_TARGET_UNIX)
     add_compile_options(-Wno-unknown-warning-option)
 
     if (NOT CLR_CMAKE_TARGET_ANDROID)
-        execute_process(COMMAND  brew --prefix OUTPUT_VARIABLE brew_prefix OUTPUT_STRIP_TRAILING_WHITESPACE)
-        set(ICU_HOMEBREW_INC_PATH "${brew_prefix}/opt/icu4c/include")
+        if(CLR_CMAKE_TARGET_OSX)
+            execute_process(COMMAND  brew --prefix OUTPUT_VARIABLE brew_prefix OUTPUT_STRIP_TRAILING_WHITESPACE)
+            set(ICU_HOMEBREW_INC_PATH "${brew_prefix}/opt/icu4c/include")
+        endif()
```

Comment:
>endif [](start = 8, length = 5)

should endif be removed and also remove the `if(CLR_CMAKE_TARGET_OSX)` in line 28 so the whole block be for OSX?

Reply:
If we guard the entire block by OSX, we lose finding out on finding `unicode/utypes.h` on Linux, right? Is `unicode/utypes.h` needed on Linux?

Reply:
but I am seeing it is searching in the ICU_HOMEBREW_INC_PATH path. no?

---
In reply to: [599202326](https://github.com/dotnet/runtime/pull/50069#discussion_r599202326) [](ancestors = 599202326)

Reply:
Huh. I tried it out. It builds :confused:

Reply:
> but I am seeing it is searching in the ICU_HOMEBREW_INC_PATH path. no?

Not quite, [find_path](https://cmake.org/cmake/help/latest/command/find_path.html)'s `PATH` argument is an *additional* list of paths to search:

> HINTS, PATHS
>
>    Specify directories to search in addition to the default locations. The ENV var sub-option reads paths from a system environment variable.